execute_pipeline.py

```diff
@@ -82,6 +106,18 @@ def _CreateArgumentParser():
       type=str,
       default="..",
       help="Root directory where the input, output, and tool repositories live")
+  parser.add_argument(
+      "--local_repo",
+      type=str,
+      default=None,
+      help="Directory where local proto and gapic configs lives. This flag is "
+           "used for remote execution only.")
+  parser.add_argument(
+        "--env",
+        type=str,
+        default=None,
+        help="Specify which environment should be used to execute pipeline. "
+             "Valid values are 'LOCAL' and 'REMOTE'")
```

Comment:
The spec of this flag is still unclear to me. Please add more comments on help as well.  What I care are:

- What will happen if it's not specified (`None`)?  It clearly doesn't satisfy `if flags.env` in line 146, so it does not sett up `pipeline_id`, but `env != 'local'` is true at line 60 so it tries to run the remote pipeline. Is it the expected behavior?
- The case sensitiveness -- the help string says 'LOCAL' and 'REMOTE', but they are case-insensitive actually because you invoke tolower(). Please say they are case-insensitive, I prefer lowercase `'local'` and `'remote'` for the example values (if this is going to be case-sensitive after you made other environment -- then I think these "local" and "remote" also should be case-sensitive, it's less confusing).